libsolidity/formal/CHC.cpp

```diff
@@ -541,6 +541,79 @@ void CHC::endVisit(Return const& _return)
 	m_currentBlock = predicate(*returnGhost);
 }
 
+bool CHC::visit(TryStatement const& _tryStatement)
+{
+	FunctionCall const* externalCall = dynamic_cast<FunctionCall const*>(&_tryStatement.externalCall());
+	solAssert(externalCall && externalCall->annotation().tryCall, "");
+
+	solAssert(m_currentFunction, "");
+	auto const& functionBody = m_currentFunction->body();
+
+	auto tryHeaderBlock = createBlock(&_tryStatement, PredicateType::FunctionBlock, "try_header_");
+	auto tryAfterCallSuccessBlock = createBlock(&_tryStatement, PredicateType::FunctionBlock, "try_after_call_success_");
+	auto tryAfterCallFailBlock = createBlock(&_tryStatement, PredicateType::FunctionBlock, "try_after_call_fail_");
+	auto afterTryBlock = createBlock(&functionBody, PredicateType::FunctionBlock);
+
+	connectBlocks(m_currentBlock, predicate(*tryHeaderBlock));
+	setCurrentBlock(*tryHeaderBlock);
+	// fine-grained control over how the external call is visited
+	// cannot use "_tryStatement.externalCall().accept(*this);"
+	auto const* call = dynamic_cast<FunctionCall const*>(&_tryStatement.externalCall());
```

Comment:
Isn't this the same as `externalCall` declared at the beginning?

Reply:
Yes, it is, I completely missed that :)